hackage-server.cabal

```diff
@@ -102,11 +102,8 @@ flag network-uri
   description: Get Network.URI from the network-uri package
   default: True
 
-executable hackage-server
-  if ! flag(build-hackage-server)
-    buildable: False
-  main-is: Main.hs
-  other-modules:
+library
```

Comment:
Only if you're all OK with only being buildable with an unreleased version of Cabal :O